### PR TITLE
Registration Page:FE

### PR DESCRIPTION
**Pull request by lyudad**

### CODE REVIEW

**Comment by GabisPapin** [comment]
Registration-page

File: src/pages/Registration.styles.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+export const Div3 = styled.div`
+display: flex;
+text-align: center;
+justify-content: space-around;
+margin-bottom: 50px;`
+
+export const P = styled.p`
+text-align: center;
+font-size: 16px;
```
Comment (lyudad) [inline src/pages/Registration.styles.ts:26]: all fons should be in constants

File: src/pages/Registration.styles.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+color: rgba(116, 114, 114, 0.932);
+margin-bottom: 30px;`
+
+export const Button = styled.button`
+&:hover {
+  background: rgb(33, 163, 33);
+  color: white;
```
Comment (lyudad) [inline src/pages/Registration.styles.ts:33]: all colors should be in constants

File: src/pages/RegistrationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+const RegistrationPage: FC = () => {
+  return (
+    <Div1>
+      <H1>Complete your registration</H1>
+      <P>Please select your role</P>
+      <Div2>
+        <P>I want to</P>
+        <Div3>
+          <Button>Work as a freelancer</Button>
```
Comment (lyudad) [inline src/pages/RegistrationPage.tsx:12]: no, we connected i18n, in the layout, we should not use pure text only from constant

**Comment by GabisPapin** [comment]
added all text in react-i18next and added colors and fonts in constants. Obs: Didn't change anything in design page.

**Comment by GabisPapin** [comment]
corrected the style file. Obs: didn't change the screen design.

File: src/pages/Registration.styles.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { backgroundColor, buttonColorHover, Color, FontSize } from '../style/App.styled';
```
[the remark this answers is not in the document]
Comment (GabisPapin) [reply src/pages/Registration.styles.ts]: I changed it

**Comment by GabisPapin** [comment]
created constants/styles and put all color and font size. Obs: Didn't change the screen design.

**Pull request merged**